kernel/float64.mli

```diff
@@ -19,9 +19,19 @@ val is_nan : t -> bool
 val is_infinity : t -> bool
 val is_neg_infinity : t -> bool
 
-val to_string : t -> string
 val of_string : string -> t
 
+(** Print a float exactly as an hexadecimal value (exact decimal
+ * printing would be possible but sometimes requires more than 700
+ * digits). *)
+val to_string : t -> string
```

Comment:
I think I'd have preferred not to change the semantics of `Float64.to_string` and introduce `Float64.to_hex_string` instead of renaming the decimal version. WDYT?

Reply:
Indeed, here it is.